### PR TITLE
Fix ping pong min max price

### PR DESCRIPTION
**Pull request by vitaliystoliarovcc**
Asana ticket: https://app.asana.com/0/1125859137800433/1200206056851195/f

Changed table structure to fix overwriting of items with the same Ping price

Depends on: https://github.com/bitfinexcom/bfx-hf-algo/pull/165 (should be merged last)

### CODE REVIEW

File: lib/ping_pong/events/life_start.js [hunk 1]
```diff
@@ -53,7 +54,7 @@ const onLifeStart = async (instance = {}) => {
   }
 
   // Handle saved pongs
-  const pongOrders = Object.keys(activePongs).map(price => (
+  const pongOrders = getPings(activePongs).map(price => (
```
Comment (robertkowalski) [inline lib/ping_pong/events/life_start.js:57]: not getPongs?

File: lib/ping_pong/util/ping_pong_table.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  return [
+    ...pingPongTable,
+    [preparePrice(pingPrice), preparePrice(pongPrice)]
+  ]
+}
+
+const extract = (pingPongTable, price) => {
+  const pingItem = pingPongTable.find(item => item[0] === price)
+  const pongPrice = pingItem[1]
+  const nextPingPongTable = pingPongTable.filter(item => item !== pingItem)
```
Comment (robertkowalski) [inline lib/ping_pong/util/ping_pong_table.js:15]: hmm

```
[] !== []
true
[1] !== [2]
true
[1] !== [1]
true
```

File: lib/ping_pong/events/life_start.js (hunk 1 above)
Comment (vitaliystoliarovcc) [reply lib/ping_pong/events/life_start.js:57]: according to the previous logic, it should extract values ​​from the first column (previously it was a key)

**Review by avsek477 — APPROVED** [review]
lgtm

**Review by tarcisiozf — APPROVED** [review]
lgtm

**Pull request merged**